eve/workers/pod.yaml

```
@@ -86,7 +86,7 @@ spec:
         - -ec
         - |
           sleep 10 # wait for mongo
-          /usr/src/app/docker-entrypoint.sh npm start | tee -a /artifacts/s3.log
+          /usr/src/app/docker-entrypoint.sh npm run management_agent | tee -a /artifacts/management.log
```

If you want to generate a new UUID on every build it can possibly be do as an early step of the aggressor that runs `echo "export UUID_VAR=<newly-generated-uuid> > /artifacts/uuid` then you can do something like this in the management pod:
```suggestion
          source /artifacts/uuid
          /usr/src/app/docker-entrypoint.sh npm run management_agent | tee -a /artifacts/management.log
```